### DISCUSSION TITLE
Code Project as a subscription model

### DISCUSSION THREAD

**Discussion by michaelurge** [codeproject/CodeProject.AI-Server#162] [General]
After recently reading about the financial status of code project, and its valuable resources...have you considered a subscription paid model for use of the resources and ongoing development in projects, especially the AI aspect.?  Myself, among many different users wouldn't mind paying a few hundred bucks a year for this?

**Comment by alliasx** [comment]
Yes I believe that should be implemented as well. Or at least set up a donate page to keep the project better funded.

**Comment by ajnienaber** [comment]
In the meantime, is there a way to send financial contributions for this valuable project?

**Comment by Dmeyer07** [comment]
Speak for yourself.. Opening this project up for donations is one thing.. Requiring a monthly or yearly subscriptions (not to mention one that is "hundreds of dollars" as you requested) is another.  If you want to see this project fall flat on its face, yes go ahead require a yearly subscription and watch how quick everyone leaves.  If financial assistance is needed, setup somewhere to donate.
